test/test_sync.cpp

```diff
@@ -3783,6 +3771,76 @@ TEST(Sync_UploadDownloadProgress_7)
     // down the session that is in the process of being created.
 }
 
+TEST(Sync_UploadProgress_EmptyCommits)
```

Comment:
This test doesn't pass on master, but I think the new behavior is sensible.

Reply:
There seems to be no issue with the test